website/www/site/content/en/documentation/programming-guide.md

```diff
@@ -4623,7 +4628,7 @@ either when I receive 50 elements, or every 1 second".
 ### 9.4. Setting a trigger {#setting-a-trigger}
 
 When you set a windowing function for a `PCollection` by using the
-<span class="language-java">`Window`</span><span class="language-py">`WindowInto`</span>
+<span class="language-java">`Window`</span><span class="language-py">`WindowInto`</span><span class="language-go">`WindowInto`</span>
```

Comment:
In this case, to make it look more Go, clarify directly that it's `beam.WindowInto` since that's how users will always see it.

Note: Fun feature of CSS: If two blockers *would* have the same content, you can just put two classes in the span class argument. eg. `<span class="language-py language-go">.